### PR TITLE
[cypress][SIMS #1549] Institution Programs - Adding Test Cases

### PR DESCRIPTION
**Pull request by velang01**
- Adding Test Cases

### CODE REVIEW

File: testing/src/cypress/page-objects/Institution-objects/DashboardInstitutionObject.ts [hunk 1]
```diff
@@ -49,6 +49,21 @@ export enum CourseLoadCalculationOptions {
...
+export enum IncludeMinimum15InstructionalHours {
+  Yes = "yes",
+  No = "no",
+}
+
+export enum IsAviationProgram {
+  Yes = "yes",
+  No = "no",
+}
+
```
Comment (andrepestana-aot) [inline testing/src/cypress/page-objects/Institution-objects/DashboardInstitutionObject.ts:66]: Enum should in capitals.

File: testing/src/cypress/page-objects/Institution-objects/DashboardInstitutionObject.ts [hunk 2]
```diff
@@ -94,6 +109,16 @@ export enum TravelProgramEligibilityOptions {
...
+export enum HasInternationalExchangeOptions {
+  Yes = "yes",
+  No = "no",
+}
+
+export enum HasInternationalExchangeProgramEligibilityOptions {
+  Yes = "yes",
+  No = "no",
+}
+
```
Comment (andrepestana-aot) [inline testing/src/cypress/page-objects/Institution-objects/DashboardInstitutionObject.ts:121]: Enum should in capitals.

File: testing/src/cypress/e2e/02-institution/04-manage-location.cy.ts [hunk 3]
```diff
@@ -588,7 +588,7 @@ describe("Manage Location", () => {
   });
 });
 
-describe("Add New Location and update for the institution", () => {
+describe.skip("Add New Location and update for the institution", () => {
```
Comment (dheepak-aot) [inline testing/src/cypress/e2e/02-institution/04-manage-location.cy.ts:591]: Comment for skip?

File: testing/src/cypress/e2e/02-institution/06-manage-users.cy.ts [hunk 4]
```diff
@@ -86,7 +86,7 @@ describe("Manage Users - Add new user", () => {
       .should("have.text", "Admin");
     manageUsersObject
       .isLegalSigningAuthorityButton()
-      .should("have.text", "Legal signing authority");
+      .should("have.text", "Legal Signing Authority");
```
Comment (dheepak-aot) [inline testing/src/cypress/e2e/02-institution/06-manage-users.cy.ts:89]: Is it case sensitive?

File: testing/src/cypress/e2e/02-institution/07-location-programs.cy.ts [hunk 5]
```diff
@@ -12,6 +29,10 @@ const institutionHelperActions = new InstitutionHelperActions();
 const programDetailViewObject = new ProgramDetailViewObject();
 
 describe("Location Program", () => {
+  // This will be replaced if necessary with the program created by the data seeds
```
Comment (dheepak-aot) [inline testing/src/cypress/e2e/02-institution/07-location-programs.cy.ts:32]: 👍

File: testing/src/cypress/e2e/02-institution/06-manage-users.cy.ts (hunk 4 above)
Comment (velang01) [reply testing/src/cypress/e2e/02-institution/06-manage-users.cy.ts:89]: Yeah the string validation is case sensitive.

**Comment by dheepak-aot** [comment]
Great work @velang01. Highly appreciate your tenacity towards work.

File: testing/src/cypress/e2e/02-institution/07-location-programs.cy.ts [hunk 6]
```diff
@@ -166,7 +183,153 @@ describe("Location Program", () => {
...
+    );
+    Object.values(DeliveredOnlineAlsoOnsiteOptions).forEach((option) => {
+      dashboardInstitutionObject.deliveredOnlineAlsoRadioButton(option);
+    });
+  });
+
+  it("Verify that there are two options (yes and no) for credits earned when course delivered on-site question", () => {
+    dashboardInstitutionObject.deliverabilityOnsiteRadioButton(
+      ProgramDeliveryOptions.Online
+    );
```
[the remark this answers is not in the document]
Comment (velang01) [reply testing/src/cypress/e2e/02-institution/07-location-programs.cy.ts:210]: typo fixed it

**Review by dheepak-aot — APPROVED** [review]
Thanks for making the changes. LGTM 👍

**Review by ann-aot — APPROVED** [review]
Good work @velang01 👍

**Pull request merged**